bindata/etcd/cluster-restore.sh

```diff
@@ -132,6 +165,7 @@ cp -p "${SNAPSHOT_FILE}" "${ETCD_DATA_DIR_BACKUP}"/snapshot.db
 # Extract and start restore-etcd pod.yaml
 if ! extract_and_start_restore_etcd_pod; then
   # If backup doesn't have restore-pod.yaml, copy it from local disk
+  echo "Warning: could not extract restore pod. Using the restore pod manifest on the disk."
   cp -p ${RESTORE_ETCD_POD_YAML} ${MANIFEST_DIR}/etcd-pod.yaml
```

Comment:
do we check that $RESTORE_ETCD_POD_YAML exists?

Reply:
uncontrolled failures are where we get into trouble can we make sure these cases are covered?

Reply:
For example is it possible that someone could try to restore to a new node vs an existant.

Reply:
OK. I can add a check for $RESTORE_ETCD_POD_YAML.

Reply:
What would be the difference between a new node and an existing node? The restore script would appear to have a hard dependency on the etcd env having been written to the node's fs in either case.

Reply:
OK fair it was a general question is it handled?